docs/http/http-spans.md

```diff
@@ -176,14 +175,6 @@ The attribute value MUST consist of either multiple header values as an array of
 
 **[8]:** The value SHOULD be normalized to lowercase.
 
-Consider always setting the transport when setting a port number, since
-a port number is ambiguous without knowing the transport, for example
-different processes could be listening on TCP port 12345 and UDP port 12345.
-
-**[9]:** If not default (`tcp` for `HTTP/1.1` and `HTTP/2`, `udp` for `HTTP/3`).
-
-**[10]:** The value SHOULD be normalized to lowercase.
-
```

Comment:
In case other reviewers wonder about the weird diff: The first two entries are removed, and the third one is now [8] but applies to network.type instead.